### PR TITLE
Remove usage of deprecated JArrayHelper from com_content

### PR DESCRIPTION
**Pull request by izharaazmi**
This is one from the series of edits in which we try to remove usage of deprecated JArrayHelper from the entire Joomla core system. Taking one extension/library at a time for easy tests.

### CODE REVIEW

File: components/com_content/models/articles.php [hunk 1]
```diff
@@ -409,8 +410,6 @@ protected function getListQuery()
 
 			if (!empty($first))
 			{
-				JArrayHelper::toString($authorAlias);
```
Comment (izharaazmi) [inline components/com_content/models/articles.php]: Removed this line as it was having no effect, return value was never used.

**Comment by wmchris** [comment]
I have tested this item :red_circle: unsuccessfully on c05130d2da1108cdbfe2aee174bc88e4499fd00e<br /><br />Results in error after publishing a new article (in my test scenario as featured)
Errormessage<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8396">issues.joomla.org/joomla-cms/8396</a>.</sub>

**Comment by wmchris** [comment]
sorry forgot to add @icampus in last commit<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8396">issues.joomla.org/joomla-cms/8396</a>.</sub>

**Comment by izharaazmi** [comment]
Since this fix was done with another PR that is already merged, i'm closing this.

**Pull request closed without merging**